### PR TITLE
Implement start page, refs: #54527

### PR DESCRIPTION
**Pull request by TheWalkingLeek**

### CODE REVIEW

File: frontend/app/routes/teams/index.js [hunk 1]
```diff
@@ -34,6 +34,16 @@ export default class TeamsIndexRoute extends Route {
     }
   }
 
+  beforeModel(transition) {
+    let definedParamValues = Object.values(transition.to.queryParams).filter(
+      value => !!value
```
Comment (lucashabersaat) [inline frontend/app/routes/teams/index.js:39]: use isPresent?

File: frontend/app/templates/components/nav-bar.hbs [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
   <div class="container">
-    <div class="navbar-header">
-      {{navbar.toggle}}
-      {{#link-to "index" class="navbar-brand"}}Cryptopus{{/link-to}}
-    </div>
     <navbar.content>
       <div class="form-inline">
-        <Input @class="form-control mr-sm-2" @type="search" placeholder={{t "search.index.type_to_search"}} />
+        {{#if this.isStartpage}}
+          <Input @class="form-control mr-sm-2" @type="search" placeholder={{t "search.index.type_to_search"}} {{autofocus}} />
```
Comment (lucashabersaat) [inline frontend/app/templates/components/nav-bar.hbs:18]: Using cryptopus is so fast now. :speedboat:

File: frontend/app/templates/components/side-nav-bar.hbs [hunk 3]
```diff
@@ -1,35 +1,55 @@
...
-            {{folder.name}}
+<div id="sidebar">
+  <nav class="navbar navbar-dark navbar-inverse">
+    <div class="navbar-header">
+      <LinkTo @route="index">
+        <img src="ember/assets/images/cryptopussy_without_text.svg" alt="logo" class="logo-icon">
+        <span class="navbar-brand">Cryptopus</span>
+      </LinkTo>
+    </div>
+    <div class="navbar navbar-collapse nav-side-bar" id="nav-side-bar-container" {{did-insert this.setupModal this}}>
```
Comment (lucashabersaat) [inline frontend/app/templates/components/side-nav-bar.hbs:9]: I am to blame, I know. However, `{{did-insert this.setupModal this}}` can be dropped?
Together with the function in `side-nav-bar.js`?

File: config/locales/en.yml [hunk 4]
```diff
@@ -43,6 +43,7 @@ en:
   new: New
   save: Save
   close: Close
+  select_team: Please select a team or use the search
```
Comment (lucashabersaat) [inline config/locales/en.yml:46]: Thinking about it...maybe something less imperative?
I like: _Looking for a password?_ (similar to before, actually)
Of course, if you change it, don't forget to change it for all languages, duh.

**Comment by lucashabersaat** [comment]
Don't forget to fix the corresponding specs. I switched them to skipped.

File: frontend/app/routes/teams/index.js (hunk 1 above)
Comment (TheWalkingLeek) [reply frontend/app/routes/teams/index.js:39]: Thought the same, checked their docs, doesn't work.

**Comment by psunix** [comment]
focus to search input field when ?

or maybe add a second search input field on start page ? one with placeholder ? then we could get rid of lots of long text ?

**Pull request merged**